LanguageExt.Core/Effects/Schedule/Schedule.cs

```diff
@@ -13,9 +13,9 @@ namespace LanguageExt;
 /// of schedulers and then union them using `|` or intersect them using `&amp;`.  Union will take the minimum of the two
 /// schedules to the length of the longest, intersect will take the maximum of the two schedules to the length of the shortest.
 ///
-/// Any IEnumerable&lt;Duration&gt; can be converted to a Schedule using ToSchedule() or Schedule.FromDurations(...).
-/// Schedule also implements IEnumerable&lt;Duration&gt; so can make use of any transformation on IEnumerable.
-/// A Schedule is a struct so an AsEnumerable() method is also provided to avoid boxing.
+/// Any `IEnumerable&lt;Duration&gt;` can be converted to a Schedule using `ToSchedule()` or `Schedule.FromDurations(...)`.
+/// Schedule also implements `IEnumerable&lt;Duration&gt;` so can make use of any transformation on IEnumerable.
+/// A Schedule is a struct so an `AsEnumerable()` method is also provided to avoid boxing.
```

Comment:
You can use the `<` and `>` characters instead of `&lt;` and `&gt;`.  The [API doc-gen tool](https://github.com/louthy/best-form) is smart enough to know what are tags and what's pure documentation, and deals with them correctly.  Never been a fan of putting these special XML characters in to documentation read by programmers, it completely blinds the reader.

https://louthy.github.io/language-ext/LanguageExt.Core/index.html

Reply:
Done